gremlin-go/driver/connection_test.go

```diff
@@ -29,22 +50,22 @@ func TestConnection(t *testing.T) {
 			assert.NotNil(t, resultSet)
 			result := resultSet.one()
 			assert.NotNil(t, result)
-			assert.Equal(t, result.AsString(), "[0]")
+			assert.Equal(t, "[0]", result.AsString())
```

Comment:
Shall we improve best practices around testing by adding messages to asserts? If tests get broken, it is easier to find the root cause from the test report if additional context is provided. Example from: https://pkg.go.dev/github.com/stretchr/testify/assert#Equal
`assert.Equal(t, a, b, "The two words should be the same.")`